packages/dap/src/hpkeConfig.spec.ts

```diff
@@ -51,29 +51,35 @@ describe("DAP HpkeConfigList", () => {
   });
 
   it("selects the first config that it recognizes", () => {
+    const list = HpkeConfigList.parse(
+      Buffer.from(
+        "0022" + // Overall length prefix
+          // Valid configuration first
+          "ff" + // id
+          "0010" + // kem_id, KemId.DhkemP256HkdfSha256
+          "0001" + // kdf_id, KdfId.HkdfSha256
+          "0001" + // aead_id, AeadId.Aes128Gcm
+          "0008" + // Length prefix for public_key
+          "4141414141414141" + // public_key
+          // Unrecognized configuration
```

Comment:
I believe should be in the other order to test that we're not just taking the first one

Reply:
Whoops, good point, didn't realize I had flipped them.